example/methods_directory/cg_diag.json

```diff
@@ -0,0 +1,64 @@
+{
+    "name": "Conjugate gradient diagonalization method",
+    "flowchartId": "none",
+    "methodGraph": [
+        {
+            "name": "Conjugate Gradient Method",
+            "type": "unit-method",
+            "flowchartId": "um00001",
+            "next": "um00002",
+            "head": true
+        },
+        {
+            "name": "Davidson diagonalization",
+            "type": "unit-method",
+            "flowchartId": "um00002"
+        }
+    ],
+    "units": [
+        {
+            "flowchartId": "um00001",
+            "methodPath": "mathematical/opt/diff/ordern/cg",
+            "tier1": {
+                "name": "Optimization",
+                "slug": "opt"
+            },
+            "parameters": {
+                "maximumNumberOfIterations": {
+                    "value": 50,
+                    "precision": false
+                },
+                "convergenceThreshold": {
+                    "value": 1e-8,
+                    "precision": true
+                }
+            }
+        },
+        {
+            "flowchartId": "um00002",
+            "methodPath": "mathematical/linalg/diag/davidson",
+            "tier1": {
+                "name": "Linear Algebra",
+                "slug": "linalg"
+            },
+            "parameters": {
+                "nEigval": {
```

Comment:
Should be full names, I guess, like in https://github.com/Exabyte-io/esse/pull/210/files#diff-d13e8de21146d779fbf6293b238b528a854bff3abc1278c8bda90868c3a54fc0R21